### DISCUSSION TITLE
NesMaker

### DISCUSSION THREAD

**Discussion by silverfox0786** [sanni/cartreader#75] [Ideas & Suggestions]
Do you think it would be possible to get NesMaker Cart Support

Namely this one

https://www.tindie.com/products/mrtentacle/nesmaker-compatible-nes-pcb/

**Comment by sanni** [comment]
Doesn't look too different from the already supported INL NesMaker cart, both using the same SST 39SF040 chip. So it might just work.

**Reply by silverfox0786** [reply]
Ah right. I didn't realise INL was supported. I would guess it would work then, because these PCBs are programmable using the INL programmer. 

Thanks
